watson-developer-cloud/discovery-components
# fix: keep aggregationResults in SearchContext

#### What do these changes do/fix?
This updates to add `aggregationResults` back into the `SearchContext`. This keeps it in sync with the `globalAggregationsResponseStore`, which is replacing the `aggregationResults`. However, since this is a breaking change, we should keep supporting it until the next major version.

#### How do you test/verify these changes?
You can test against CM in Tooling. Use `const { aggregationResults } = useContext(SearchContext);` instead of grabbing the aggregation results from the global aggregations response store there and see that the aggregation results correctly load for a CM project. Test that without these updates, using `aggregationResults`, they don't.

#### Have you documented your changes (if necessary)?
Included a comment in the code. Should we document upcoming breaking changes anywhere else?

#### Are there any breaking changes included in this pull request?
